completion/available/brew.completion.bash

```diff
@@ -2,6 +2,10 @@ if which brew >/dev/null 2>&1; then
   if [ -f `brew --prefix`/etc/bash_completion ]; then
     . `brew --prefix`/etc/bash_completion
   fi
+  
+  if [ -f `brew --prefix`/etc/bash_completion.d/brew ]; then
```

Comment:
This shouldn't be necessary to do, since the above script (`etc/bash_completion`) is already looping through the `etc/bach_completion.d`  directory and sourcing all of the files from that directory. Specifically including one of them is not the right way to do this. If you have the `etc/bash_completion` script, it should already take care of that automatically.